sequelize/migrations/correlation/04_correlations.js

```diff
@@ -0,0 +1,121 @@
+'use strict';
+
+// See manual:
+// https://sequelize.readthedocs.io/en/latest/docs/migrations/#functions
+
+// CHANGE NOTES:
+// Changes between 1.2.0 and 2.0.0:
+// - Added new column: parentProcessInstanceId
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+
+    console.log('Running updating migrations');
+
+    const correlationTableInfo = await queryInterface.describeTable('Correlations');
+
+    const tableHasMatchingColumns =
+      correlationTableInfo.state !== undefined
+      && correlationTableInfo.error;
```

Comment:
```suggestion
      && correlationTableInfo.error !== undefined;
```

Just to keep things in sync.